coins/src/adapters/moneyMarkets/euler/eulerV2.ts

```diff
@@ -106,7 +115,11 @@ function formWrites(
     const coinData: CoinData | undefined =
       underlyingPrices[m.underlying.toLowerCase()];
     const rate = m.sharePrice;
-    if (coinData == null || rate == null) return;
+    if (coinData == null || rate == null || !m.amount) return;
+
+    const tvl = m.amount * coinData.price / 10 ** coinData.decimals;
+    if (tvl < 1e5) return 
```

Comment:
add a comment that we are filtering out tokens under 10k, also `return;` to be safe, else it could run the next line